src/frameworks/next/index.ts

```diff
@@ -234,7 +256,16 @@ export async function build(dir: string): Promise<BuildResult> {
     console.log("");
   }
 
-  return { wantsBackend, headers, redirects, rewrites, trailingSlash };
+  const i18n = nextjsI18n && !nextjsI18n.domains ? nextjsI18n : undefined;
```

Comment:
Nit: Maybe add parens to make this easier to read?

`  const i18n = (nextjsI18n && !nextjsI18n.domains) ? nextjsI18n : undefined;`

Reply:
@TheIronDev prettier does not allow parenthesis there. Maybe a variable name could help? e.g:
```ts
  const isUsingI18nButNotDomainRouting = nextjsI18n && !nextjsI18n.domains;
  const i18n = isUsingI18nButNotDomainRouting ? nextjsI18n : undefined;
```